mqtt_base_block.py

```diff
@@ -15,8 +15,8 @@ class MqttBase(Block):
     topic = StringProperty(title="Topic", default="", allow_none=False)
 
     def __init__(self):
-        self.client = mqtt
         super().__init__()
+        self.client = None
```

Comment:
This is debatable, but I would call this `self._client`. Not sure that @mattdodge would agree, but I think he would at least not entirely disagree.